msal/tests/dev apps/DesktopTestApp/MainForm.cs

```diff
@@ -349,6 +356,11 @@ private UIBehavior GetUIBehavior()
                 behavior = UIBehavior.Consent;
             }
 
+            if (noPrompt.Checked)
+            {
+                behavior = UIBehavior.NoPrompt; ;
```

Comment:
double ;

Reply:
nice catch! thank you!

---
In reply to: [237127805](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1424#discussion_r237127805) [](ancestors = 237127805)